packages/ui-kit/src/components/Leaderboard/Leaderboard.tsx

```diff
@@ -80,55 +81,11 @@ export const LeaderboardComponent = React.forwardRef<HTMLDivElement, Leaderboard
         const values =
           data.rows?.map((row) => (row[row.length - 1] === null ? null : Number(row[row.length - 1]))) || []
 
-        const customLeaderboardChartLabelsPlugin: Plugin<'bar'> = {
```

Comment:
This is just factored out into `getCustomChartLabelsPlugin`? Any other changes?

Reply:
That is all for Leaderboard component